boardswarm-cli/src/main.rs

```diff
@@ -386,6 +386,54 @@ struct BmapWriteArgs {
     file: PathBuf,
 }
 
+#[derive(Clone, Debug)]
+enum VolumeArg {
+    Id(u64),
+    Name(String),
+}
+
+impl VolumeArg {
+    async fn volume(&self, mut client: Boardswarm) -> Result<Option<u64>, anyhow::Error> {
```

Comment:
This never seems to return `Ok(None)` so can just be `Result<u64, ...>` i guess.. Ok i guess this should be `lookup_volume` or `find_volume` or somesuch just to make it more clear what it does